README.md

```diff
@@ -161,11 +161,9 @@ Changes not staged for commit:
 In this example, `README.md` has conflicts. Open that file and look for the following:
 
 ```
-<<<<<<< HEAD
-some code
-=========
+
 some code
->>>>>>> new-commit
+
```

Comment:
These lines shouldn't have been edited - the rest looks good. If you can add these back in, we're ready to go.